src/ast/mod.rs

```diff
@@ -2754,7 +2756,8 @@ impl fmt::Display for Statement {
                         }
                     }
                 }
-                if *external {
+                // BigQuey doesn't have this external format
+                if *external && file_format.is_some() && location.is_some() {
```

Comment:
external tables not always have `STORED AS ...` format (eg. BigQuery). Adding condition to check if there is file_format and location. Otherwise it was raising an error about unwrap on None.